py-polars/polars/internals/frame.py

```diff
@@ -3528,8 +3551,34 @@ def melt(
         value_vars
             Values to use as identifier variables.
 
-        Returns
-        -------
+        Examples
+        --------
```

Comment:
Could you add a line between the `Examples` and the start of the code snippets?

Reply:
Sure. Does this apply to all `Examples` headings?

Reply:
Yeah.. It is a recommendation of numpy doc. We don't check it in CI, so here is the manual nit. :P

Reply:
Ok, added the lines :D